src/components/Contact.tsx

```
@@ -0,0 +1,136 @@
+"use client";
+
+import { useState } from "react";
```

ESLint

[error] 'useState' is defined but never used.

(@typescript-eslint/no-unused-vars)